docs/native-modules-android.md

```diff
@@ -516,6 +516,15 @@ private void sendEvent(ReactContext reactContext,
      .getJSModule(DeviceEventManagerModule.RCTDeviceEventEmitter.class)
      .emit(eventName, params);
 }
+@ReactMethod
+public void addListener(String eventName) {
+ // Keep: Required for RN built in Event Emitter Calls.
```

Comment:
I think it's a good idea to add these required methods in the examples, but I'd modify the comment to be more useful for people reading it.

In iOS we have the same (https://reactnative.dev/docs/native-modules-ios#sending-events-to-javascript):
```
@implementation CalendarManager
{
  bool hasListeners;
}

// Will be called when this module's first listener is added.
-(void)startObserving {
    hasListeners = YES;
    // Set up any upstream listeners or background tasks as necessary
}

// Will be called when this module's last listener is removed, or on dealloc.
-(void)stopObserving {
    hasListeners = NO;
    // Remove upstream listeners, stop unnecessary background tasks
}

- (void)calendarEventReminderReceived:(NSNotification *)notification
{
  NSString *eventName = notification.userInfo[@"name"];
  if (hasListeners) { // Only send events if anyone is listening
    [self sendEventWithName:@"EventReminder" body:@{@"name": eventName}];
  }
}
```